src/component-library/theme/theme.kintsugi.css

```diff
@@ -1,6 +1,7 @@
 @import url('theme.base.css');
 
 :root {
+  --colors-blue: #041333;
```

Comment:
Should I ask Jay for the colors for interlay app?

Reply:
Yes - from next week all features will need to support both themes, so we'll need to specify the interlay properties before merging. (Hasn't been the case up until now, as we haven't fully released it yet).